### DISCUSSION TITLE
Which Kratos SDK should i use for JS?

### DISCUSSION THREAD

**Discussion by halvardssm** [ory/.github#16] [Q&A]
Hi! I am using the generated SDK client for Kratos ([@ory/kratos-client](https://www.npmjs.com/package/@ory/kratos-client)), but I stumbled over another package that is seemingly identical ([@ory/client](https://www.npmjs.com/package/@ory/client)). Which package should I preferably use, and what is the difference between them?

**Comment by aeneasr** [comment]
This is a great question, and sorry for the late reply. Currently, both SDKs are supported and work. We're still trying to figure out the best way to supply these SDKs for both ecosystems.
